nix/build/release-binaries.nix

```diff
@@ -18,55 +21,83 @@ in [
   }
   {
     name = "octez-admin-client";
-    description = "Administration tool for the node";
+    description = ''
+      CLI adminitrator tool with additional possibilities.
```

Comment:
It's not really clear with additional possibility to what?

I'd rather change it to "CLI administrator tool for peers management" since, according to the docs, most of its capabilities is related to peers.

What do you think?